tools/mason/MasonNew.chpl

```diff
@@ -46,20 +46,30 @@ proc masonNew(args: [] string) throws {
                                  opts=["--name"]);
 
   var showFlag = parser.addFlag(name="show", defaultValue=false);
+  var appFlag = parser.addFlag(name="app", defaultValue=false);
+  var libFlag = parser.addFlag(name="lib", defaultValue=false);
+  var lightFlag = parser.addFlag(name="light", defaultValue=false);
```

Comment:
Hampered by the `ArgumentParser` library missing mutually exclusive groups or defined option selection, you might want to manually check that at most one of these flags is true and exit with an error message if multiple are true.

Reply:
Added.